src/lib/common/RenderFormat.cpp

```diff
@@ -46,6 +46,7 @@ const char* renderFormatToString(RenderFormat format, bool noDefault)
   case NGSI_V2_KEYVALUES:           return "keyValues";
   case NGSI_V2_VALUES:              return "values";
   case NGSI_V2_UNIQUE_VALUES:       return "uniqueValues";
+  case NGSI_V2_CUSTOM:              return "custom";
```

Comment:
"Ngsiv2-Attrsformat: custom" should be mentioned in the "Custom Notifications" section in the NGSIv2 .apib. Could you confirm? (maybe it is in a file in this PR that I haven't check yet)